packages/core/strapi/lib/types/core/attributes/media.d.ts

```diff
@@ -1,36 +1,32 @@
-import { Attribute, ConfigurableOption, PrivateOption, RequiredOption } from './base';
-import { Media } from './common';
+import type { Attribute } from '@strapi/strapi';
 
 export type AllowedMediaTypes = 'images' | 'videos' | 'files' | 'audios';
 
-export interface MediaAttributeProperties<
+export interface MediaProperties<
   // Media Type
-  T extends AllowedMediaTypes = undefined,
+  T extends AllowedMediaTypes | undefined = undefined,
   // Multiple
   U extends boolean = false
 > {
   allowedTypes?: T;
   multiple?: U;
 }
 
-export type MediaAttribute<
+export type Media<
   // Media Type
-  T extends AllowedMediaTypes = undefined,
+  T extends AllowedMediaTypes | undefined = undefined,
   // Multiple
   U extends boolean = false
-> = Attribute<'media'> &
+> = Attribute.Attribute<'media'> &
   // Properties
-  MediaAttributeProperties<T, U> &
+  MediaProperties<T, U> &
   // Options
-  ConfigurableOption &
-  RequiredOption &
-  PrivateOption;
+  Attribute.ConfigurableOption &
+  Attribute.RequiredOption &
+  Attribute.PrivateOption;
 
-export type MediaValue<T extends boolean = false> = T extends true ? Media[] : Media;
+export type MediaValue<T extends boolean = false> = T extends true ? any[] : any;
```

Comment:
Can you explain the switch to `any`? I'm not sure I understand this one.

Reply:
We had [type Media = any](https://github.com/strapi/strapi/pull/16917/files/8d6754445ac51588b8ac967e42128114cebe646f#diff-b83d7c68820cb22a6c5b63b0345f1b3a4e76e8ab3d564adf3b546c2173781efbL10). I just made it transparent to avoid confusion until we add a better type definition of what a media is

Reply:
Ah, ok, good. That's better than being hidden 😆